modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoClientExperimentalCodegen.java

```diff
@@ -77,6 +77,7 @@ public String getHelp() {
 
     @Override
     public void processOpts() {
+        this.setDiscriminatorExplicitMappingVerbose(true);
```

Comment:
This isn't an appropriate place for this option. If someone were to extend this generator and modify the value (maybe they don't support this explicitly mapping in their templates), you're potentially overwriting their value.

Please move this to the constructor here and elsewhere. This would also allow our tooling to understand the defaults for the generator. Putting a default here, it's not known until execution time (via a "real" generate). See `hideGenerationTimestamp` in this constructor.